samples/Sentry.Samples.Console.Native/Sentry.Samples.Console.Native.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <TargetFrameworks>net8.0</TargetFrameworks>
+    <PublishAot>true</PublishAot>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!--
+    We set the SentryOrg and SentryProject globally in our root Directory.Build.props.
+    In your own project, uncomment the following block and set the values for your Sentry configuration.
+    Also note, these options do nothing if you are not authenticated.
+    See https://docs.sentry.io/platforms/dotnet/configuration/msbuild/
+    -->
+    <!--
+    <SentryUrl>...your Sentry URL if self-hosted, or omit this line if using sentry.io...</SentryUrl>
+    <SentryOrg>...your org...</SentryOrg>
+    <SentryProject>...your project...</SentryProject>
+    -->
+
+    <!--
+    After the above properties are configured, you can use the following features.
+    Uploading sources to Sentry during the build will enable Source Context in the Sentry issue details page.
+    Uploading symbols to Sentry will enable server-side symbolication (i.e. when the PDB is not present at runtime, or for AOT published programs).
+    -->
+    <SentryUploadSources>true</SentryUploadSources>
```

Comment:
Is it working? I tested this recently and didn't work (maybe didn't open an issue) but I only got sources with the .NET flag to bundle sources in the PDB

Reply:
yes, it works AFAIK
<img width="1227" alt="image" src="https://github.com/getsentry/sentry-dotnet/assets/6349682/9cc1f271-5d96-4636-affb-f6b6d008fd56">
<img width="712" alt="image" src="https://github.com/getsentry/sentry-dotnet/assets/6349682/bd571f3f-2a3c-4477-acb8-b518b9bee7d4">